### DISCUSSION TITLE
[Feature] The ability to Upgrade & Downgrade services

### DISCUSSION THREAD

**Discussion by 09han09** [Paymenter/Paymenter#252] [Ideas]
The ability to Upgrade & Downgrade services.

**Comment by finnjosten** [comment]
Not sure but this might be already planned for later and if it's not please add this feature :)

**Reply by 09han09** [reply]
Thats why the suggestion was made xD, so i could add it to the github roadmap. [Roadmap](https://github.com/orgs/Paymenter/projects/10)

**Comment by iByteGamingKyle** [comment]
Unbelievably excited for this feature baked in!
